examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,195 @@
+
+###############################################################################
+# Icepak design
+# ~~~~~~~~~~~~~
+# Insert icepak design and copy solid objects from Maxwell, modify region dimensions
+
+# desktop = m3d.desktop_class
+# ipk = desktop[[m3d.project_name, icepak_design_name]]
+
+ipk = pyaedt.Icepak(designname=icepak_design_name)
+ipk.copy_solid_bodies_from(m3d, no_pec=False)
+ipk.modeler.edit_region_dimensions([50, 50, 50, 50, 500, 500])
+
+###############################################################################
+# Map coil losses
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Map ohmic losses in coil
+
+ipk.assign_em_losses(
+    designname="1 Maxwell",
+    setupname=m3d.setups[0].name,
+    sweepname="LastAdaptive",
+    object_list=["Coil"],
+)
+
+###############################################################################
+# Boundary conditions
+# ~~~~~~~~~~~~~~~~~~~
+# Assign opening
+
+faces = ipk.modeler["Region"].faces
+face_names = [face.id for face in faces]
+ipk.assign_free_opening(face_names, boundary_name="Opening1")
+
+###############################################################################
+# Icepak mesh
+# ~~~~~~~~~~~~~~~~~~~
+# Assign mesh
+
+mesh_region = ipk.mesh.assign_mesh_region(objectlist=["Coil", "Core"])
+mesh_region.UserSpecifiedSettings = True
+mesh_region.MinGapX = "1mm"
+mesh_region.MinGapY = "1mm"
+mesh_region.MinGapZ = "1mm"
+mesh_region.MaxLevels = "2"
+mesh_region.MaxElementSizeX = "10mm"
+mesh_region.MaxElementSizeY = "10mm"
+mesh_region.MaxElementSizeZ = "10mm"
+mesh_region.update()
+
+
+###############################################################################
+# Icepak solution setup
+# ~~~~~~~~~~~~~~~~~~~
+
+solution_setup = ipk.create_setup()
+solution_setup.props["Convergence Criteria - Max Iterations"] = 200
+solution_setup.props["Flow Regime"] = "Turbulent"
+solution_setup.props["Turbulent Model Eqn"] = "ZeroEquation"
+solution_setup.props["Radiation Model"] = "Discrete Ordinates Model"
+solution_setup.props["Include Flow"] = True
+solution_setup.props["Include Gravity"] = True
+solution_setup.props["Solution Initialization - Z Velocity"] = "0.0005m_per_sec"
+solution_setup.props["Convergence Criteria - Flow"] = 0.0005
+solution_setup.props["Flow Iteration Per Radiation Iteration"] = "5"
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~~~~~
+# Plot temperature on the object surfaces
+
+surface_list = []
+for name in ["Coil", "Core"]:
+    surface_list.extend(ipk.modeler.get_object_faces(name))
+
+ipk.post.create_fieldplot_surface(
+    surface_list,
+    quantityName="SurfTemperature",
+    plot_name="Surface Temperature"
+)
+
+###############################################################################
+# Add 2-way coupling and solve the project
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ipk.assign_2way_coupling()
+ipk.analyze_setup(name="Setup1")
```

Comment:
```suggestion
###############################################################################
# Add 2-way coupling and solve the project
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

ipk.assign_2way_coupling()
ipk.analyze_setup(name=solution_setup.name)


###############################################################################
# Post-processing
# ~~~~~~~~~~~~~~~~~~~
# Plot temperature on the object surfaces

surface_list = []
for name in ["Coil", "Core"]:
    surface_list.extend(ipk.modeler.get_object_faces(name))

surf_temperature = ipk.post.create_fieldplot_surface(
    surface_list,
    quantityName="SurfTemperature",
    plot_name="Surface Temperature"
)

velocity_cutplane = ipk.post.create_fieldplot_cutplane(
        "Global:XZ",
        "Velocity Vectors"
    )

surf_temperature.export_image()
velocity_cutplane.export_image(orientation="right")
```

I would put the post-processing at the very end.
I added the velocity cut plane just to enrich the example.